sphinx/sphinx.gradle

```diff
@@ -8,12 +8,22 @@ description = 'Generates docs for the python libraries provided in Deephaven Cor
 
 evaluationDependsOn ':Integrations'
 
+tasks.register('copyPyClientWhl', Copy) {
+    dependsOn ':pyclient:buildPyClient'
+    from "$rootDir/pyclient/build/wheel/"
```

Comment:
consider replacing this and dependsOn with 
```
from(project(':pyclient').tasks.named('buildPyClient').get().outputs.files)
```
This one line describes both "I should rebuild when that task changes" and "I want to know the correct path to that task's outputs without hardcoding it here"